docs/manual-EN/2.query-language/3.language-structure/comment-syntax.md

```diff
@@ -7,15 +7,15 @@ Nebula supports four comment styles:
 * From a // sequence to the end of the line, as in the C programming language.
 * From a /* sequence to the following */ sequence. This syntax enables a comment to extend over multiple lines because the beginning and closing sequences need not be on the same line.
```

Comment:
because the beginning and closing sequences do not need to be on the same line.

Reply:
I think `need` can be used as a modal verb.

Reply:
I think `need` can be used as a modal verb.